diagnostic.md

```diff
@@ -62,8 +62,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
-
+Once I input the values (0, -5), it goes thtough the first statement and is false; becuase a is not > 0 then goes to the second statement and is false as well.
```

Comment:
The conditional statement does evaluate to false since both a > 0 and b > 0 evaluate to false.  However, the second return statement still runs even though the if-block fails.  It returns a - b, or 0 - (-5) = 5.